CONTRIBUTING.md

```diff
@@ -8,7 +8,7 @@
   - .NET desktop development
   - Visual Studio extension development.
   - Desktop development with C++
-- [Windows 10 SDK](https://dev.windows.com/en-US/downloads/windows-10-sdk)
+- [Windows 10 SDK](https://developer.microsoft.com/en-us/windows/downloads/windows-10-sdk/)
```

Comment:
I'm not if this is still needed for building. I think it's needed if you want to run all *E2E* tests though. 

I created a follow up. https://github.com/NuGet/Home/issues/10096